projects/demo/src/modules/components/mobile-calendar/mobile-calendar.template.html

```diff
@@ -144,6 +152,14 @@
             >
                 Single date or a range
             </ng-template>
+            <ng-template
+                documentationPropertyMode="input"
+                documentationPropertyName="multi"
+                documentationPropertyType="boolean"
+                [(documentationPropertyValue)]="multi"
+            >
+                Multi date or a range
```

Comment:
`Array of single dates` would be easier to understand. After refactor that should probably become a singular mode `singe | multi | range` instead of 2 booleans